### PR TITLE
Making some changes to README.md

### PR DESCRIPTION
**Pull request by sunny-verma**
Fixing Issue #88

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,12 @@
...
-1. image-patching: overcloud-full.qcow2 image patching resources
-
-2. openstack-tripleo-heat-templates: Heat-template changes required for OSP Director and Nuage integration
-
-3. generate-cms-id: CMS ID generation resources
-
-4. documentation: The details of integration, architecture and deployment steps can be found under Documentation folder
\ No newline at end of file
+1. **image-patching**: overcloud-full.qcow2 image patching resources.
+2. **nuage-tripleo-heat-templates**: Heat-template changes required for OSP Director and Nuage integration
```
Comment (psairam) [inline README.md]: can we reword it this way
Heat templates requried to for Nuage Integration with OSP Director?

File: image-patching/stopgap-script/README.md [hunk 2]
```diff
@@ -350,7 +350,16 @@ If image patching fails for some reason then remove the partially patched overcl
...
 
 
-Once the patching is done successfully copy back the patched image to /home/stack/images/ on undercloud-director   
+The image patching script uses libguestfs virt-customize function, which adds the machine-id to the overcloud-full.qcow2 resulting in all the overcloud nodes to have the same machine-id. The following command needs to be applied once the patching is done successfully to remove the machine-id from the overcloud-full.qcow2.
+ 	
+    virt-sysprep --operation machine-id -a overcloud-full.qcow2
+ 
+In order to verify that  machine-id is clear in the overcloud image, run the following command and you should see empty output:
+
+	guestfish -a overcloud-full.qcow2 run : mount /dev/sda / : cat /etc/machine-id
```
Comment (psairam) [inline image-patching/stopgap-script/README.md]: let's provide command line sample

Comment (sunny-verma) [reply image-patching/stopgap-script/README.md]: That's the cli sample itself.

File: README.md (hunk 1 above)
Comment (sunny-verma) [reply README.md]: Sure.

**Pull request merged**